.github/workflows/release-binaries.yml

```diff
@@ -17,8 +17,8 @@ on:
         required: true
         type: choice
         options:
-          - ubuntu-22.04
-          - ubuntu-22.04-arm
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
```

Comment:
For the release binaries we want to build them on the oldest possible Ubuntu version, so they are more portable.  Maybe we need a comment here explaining why.